bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -66,7 +66,7 @@
 
     private static final RecyclableArrayList.Recycler<QueueEntry> entryListRecycler =
         new RecyclableArrayList.Recycler<QueueEntry>();
-    private static final RecyclableArrayList<QueueEntry> EMPTY_ARRAY_LIST = entryListRecycler.newInstance();
+    private static final RecyclableArrayList<QueueEntry> EMPTY_ARRAY_LIST = new RecyclableArrayList<>();
 
```

Comment:
I can't remember why we can't use a simple ArrayList if we cannot recycle it

Reply:
https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java#L409

Reply:
Yes, the `ForceWriteThread` is taking a `RecyclableArrayList` so that it knows it can call `recycle()`. Otherwise we would have to check : 

```java
if (list instanceof RecyclableArrayList) {
  ((RecyclableArrayList) list).recycle();
}
```